sycl/include/sycl/platform.hpp

```diff
@@ -57,6 +62,38 @@ namespace ext::oneapi {
 class filter_selector;
 } // namespace ext::oneapi
 
+#define __SYCL_GET_INFO_BODY                                                   \
```

Comment:
Why a macro? A helper template function that performs the conversion should do just fine.

Reply:
Thanks. Done.